front/components/vaults/VaultResourcesList.tsx

```diff
@@ -103,7 +103,7 @@ const getTableColumns = ({
     meta: {
       width: "6rem",
     },
-    id: "managedBy",
+    accessorKey: "managedBy",
```

Comment:
don't you need to have an id to make columnsBreakpoints work ?

Reply:
That's what I also thought, but seems like `accessorKey` does the job and "id" as well 🤷🏼‍♂️

Reply:
Don't mind setting "id" everywhere instead of "accessorKey" if you prefer

Reply:
let's keep accessorKey only if it's the same value ..